Exiled.Events/EventArgs/LocalReportingEventArgs.cs

```diff
@@ -42,9 +42,9 @@ public LocalReportingEventArgs(Player issuer, Player target, string reason, bool
         public Player Target { get; }
 
         /// <summary>
-        /// Gets or sets the report reason.
+        /// Gets the report reason.
         /// </summary>
-        public string Reason { get; set; }
+        public string Reason { get; }
```

Comment:
This is a breaking change.

Reply:
It's against VSR modifying the reason.

Reply:
This is still a breaking change. We can add a todo comment here to address changing this in a future major release, but it's not really EXILED's job to make sure server owners don't violate VSR with plugins.

Reply:
I'm gonna implement setter in transpiler then